start_service_production_marquis.bat

```diff
@@ -0,0 +1,16 @@
+:: Script that will run all the components of our service
+:: By changing args parameters you can set up the services at arbitrary hosts and ports but remember
+:: to give the right hosts and ports in the main service command
+
+:: Activating document retrieval microservice
+start cmd /k "venv\Scripts\activate.bat && cd document_retrieval && python -m microservice.main start -H localhost -p 4322 -e production"
+
+:: Activating document similarity microservice
+start cmd /k "venv\Scripts\activate.bat && cd document_similarity && python -m microservice.main start -H localhost -p 4323 -e production"
+
+:: Activating text embedding microservice (for 1 language)
+REM start cmd /k "venv\Scripts\activate.bat && cd text_embeddings && python -m text_embedding.main start -H localhost -p 4001 -e production -mp ./data/embeddings/wiki-news-300d-1M-en.vec -ml en"
```

Comment:
What is the `REM`?

Reply:
REM is a comment in batch script. So I have easier time switching between models.

Reply:
Cool :)